.github/workflows/tests.yml

```diff
@@ -259,6 +259,7 @@ jobs:
           -e TEST_GROUP
           ghcr.io/conda/conda-ci:main-linux-python${{ matrix.python-version }}${{ matrix.default-channel == 'conda-forge' && '-conda-forge' || '' }}
           bash -c "source /opt/conda/etc/profile.d/conda.sh; \
+                   sudo /opt/conda/bin/conda install --quiet -y -c defaults --file tests/requirements.txt; \
```

Comment:
As far as I can tell the "cache docker images" system did not anticipate new test dependencies, so this is not temporary. It is pretty fast when all the dependencies are already in there though.

Reply:
Right, remember that when you change the test dependencies, [it will automatically build a new docker image for you to use during the lifetime of the pull request](https://github.com/conda/conda/blob/5afa22700009f52339f51e6f4218537d34a8dce5/.github/workflows/images.yml#L13): https://github.com/conda/conda/actions/runs/4314817101 You should be able to use that new image without having to run the conda command again (which hardcodes defaults for example).

Reply:
And yet, that system does not work. As soon as I added the flask dependency for a test repository, the checks continuously failed.